datasets/statistieken/dataset.json

```
@@ -0,0 +1,67 @@
+{
+  "type": "dataset",
+  "id": "statistieken",
+  "title": "statistieken",
+  "status": "beschikbaar",
+  "description": "Gepubliceerde statistieken conform generiek model",
+  "version": "0.0.1",
+  "owner": "Gemeente Amsterdam",
+  "publisher": "datapunt@amsterdam.nl",
+  "authorizationGrantor": "Basisstatistiek",
+  "contactPoint": {
+    "name": "Basisstatistiek",
+    "email": "basisstatistiek.ois@@amsterdam.nl"
+  },
+  "auth": "OPENBAAR",
+  "tables": [
+    {
+      "id": "measures",
+      "$ref": "measures/v1.0.0",
+      "activeVersions": {
+        "1.0.0": "measures/v1.0.0"
+      }
+    },
+    {
+      "id": "observations",
+      "$ref": "observations/v1.0.0",
+      "activeVersions": {
+        "1.0.0": "observations/v1.0.0"
+      }
+    },
+    {
+      "id": "spatialdimensions",
+      "$ref": "spatialdimensions/v1.0.0",
+      "activeVersions": {
+        "1.0.0": "spatialdimensions/v1.0.0"
+      }
+    },
+    {
+      "id": "spatialdimensionsType",
+      "$ref": "spatialdimensionsType/v1.0.0",
+      "activeVersions": {
+        "1.0.0": "spatialdimensionsType/v1.0.0"
+      }
+    },
+    {
+      "id": "temporaldimensions",
+      "$ref": "temporaldimensions/v1.0.0",
+      "activeVersions": {
+        "1.0.0": "temporaldimensions/v1.0.0"
+      }
+    },
+    {
+      "id": "temporaldimensionsType",
+      "$ref": "temporaldimensionsType/v1.0.0",
```

Volgens mij moet dit
```suggestion
      "$ref": "temporalDimensionsType/v1.0.0",
```
zijn, of die map moet hernoemd worden (laatste optie is consequenter). Amsterdam-schema is case-sensitive.